### PR TITLE
feat: Add xml_to_tree app

### PR DESCRIPTION
**Pull request by jbrundage**
Signed-off-by: Jaman Brundage <jbrundage372@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [ ] The commit is signed.
- [x] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

**Comment by jbrundage** [comment]
image

**Comment by jbrundage** [comment]
This parses any XML structure into Graph2 (for the NLP++ stuff)

File: apps/xml_to_tree/xml2.hml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    </clause>
+
+</sent>
+
+<qEOS>
+
+</qEOS>
+
+</TEXTZONE>
```
Comment (GordonSmith) [inline apps/xml_to_tree/xml2.hml:55]: Too many spaces?

Comment (jbrundage) [reply apps/xml_to_tree/xml2.hml]: Agreed, fixed.

**Review by GordonSmith — APPROVED** [review]

**Pull request merged**